.github/workflows/test-deploy.yml

```diff
@@ -0,0 +1,93 @@
+name: test-deploy
+
+on:
+  push:
+    branches:
+      [project-update-jobs]
```

Comment:
I think this can be changed to `dev`?

Reply:
If you see a PR of mine with a test-deploy.yml file, it's so that the branch in question can be deployed to dev without having to commit it to dev. The value inside on:push:branches will be the branch name, and not `dev` so that it gets deployed. I'm usually careful to remove test-deploy.yml once it's ready to merge in (I was re-testing it on dev after some rebasing).